src/Raven.Server/Documents/Handlers/SubscriptionsHandler.cs

```diff
@@ -426,6 +382,109 @@ public Task DropSubscriptionConnection()
 
             return NoContent();
         }
+
+        [RavenAction("/databases/*/subscriptions/update", "POST", AuthorizationStatus.ValidUser)]
+        public async Task Update()
+        {
+            using (ContextPool.AllocateOperationContext(out DocumentsOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                var json = await context.ReadForMemoryAsync(RequestBodyStream(), null);
+                var options = JsonDeserializationServer.SubscriptionCreationParams(json);
+
+                var id = GetLongQueryString("id", required: false);
+                SubscriptionState state;
+
+                if (id == null)
+                {
+                    state = Database.SubscriptionStorage.GetSubscriptionFromServerStore(options.Name);
+                    id = state.SubscriptionId;
+                }
+                else
+                {
+                    state = Database.SubscriptionStorage.GetSubscriptionFromServerStoreById(id.Value);
```

Comment:
I do not think we want this.

Reply:
please see the comment above 
https://github.com/ravendb/ravendb/pull/10479#discussion_r413139945